packages/core/useFileDialog/index.ts

```diff
@@ -0,0 +1,47 @@
+import { readonly, ref } from 'vue-demi'
+
+export interface Options {
+  multiple?: boolean
+  accept?: string
+}
+
+const DEFAULT_OPTIONS: Options = {
+  multiple: true,
+  accept: '*',
+}
+
+/**
+ * Open file dialog with ease.
+ *
+ * @see https://vueuse.org/useFileDialog
+ * @param options
+ */
+export function useFileDialog(options?: Partial<Options>) {
+  const files = ref<FileList | null>(null)
+
+  const openFileDialog = (localOptions?: Partial<Options>) => {
+    const _options: Options = {
+      ...DEFAULT_OPTIONS,
+      ...options,
+      ...localOptions,
+    }
+    const input = document.createElement('input')
+    input.type = 'file'
+    input.multiple = _options.multiple!
+    input.accept = _options.accept!
+
+    input.onchange = (event: Event) => {
+      const result = event.target as HTMLInputElement
+      files.value = result.files
+    }
+
+    input.click()
+  }
+
+  return {
+    files: readonly(files),
+    openFileDialog,
```

Comment:
```suggestion
    open,
```

We don't duplicate the context, and let user to do the rename when neccessary